CenterForOpenScience/osf.io
# [Hotfix][PLAT-992] Keep download count for files over multiple nodes

## Purpose

Currently when a file moves to a different node the download count is "lost", but really it's just always filtering by it's target node/preprint/whatever. This fix ensures we only filter by the file id. 

## Changes

- creates a new classmethod for page counter which filters only by action and file id
- removes now superfluous utility function `get_basic_counters`

## QA Notes

You should be able to move files from node to node and retain download counts. Also it's important to remember this works retroactively, so old files will now have old download counts from different nodes correctly added in. 

## Documentation

🐞  fix no docs.

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/browse/PLAT-992
